test/runtests.jl

```diff
@@ -3,7 +3,8 @@ using Test, Pkg
 ENV["DATADEPS_ALWAYS_ACCEPT"] = true
 ENV["JULIA_LOG_LEVEL"] = "WARN"
 
-for submodule in ["Utilities/TicToc",
+for submodule in [
+                  "Utilities/TicToc",
```

Comment:
This change looks a bit weird to me, but it's a stylistic choice...

Reply:
Ah, yes. It does look a bit funny, but I find this convenient when I want to locally test specific modules (and comment the rest). I can revert if you'd like

Reply:
Don't mind much either way, and your explanation makes sense.